src/ZMQ.jl

```diff
@@ -591,9 +591,7 @@ function __init__()
     patch = Array(Cint,1)
     ccall((:zmq_version, zmq), Void, (Ptr{Cint}, Ptr{Cint}, Ptr{Cint}), major, minor, patch)
     global const version = VersionNumber(major[1], minor[1], patch[1])
-    if version < v"3"
```

Comment:
Wouldn't hurt to leave this in `__init__` in case someone builds ZMQ.jl and then changes their libzmq installation.